src/openms/include/OpenMS/ANALYSIS/OPENSWATH/ConfidenceScoring.h

```diff
@@ -118,7 +119,7 @@ namespace OpenMS
       boost::mt19937 generator_; ///< random number generation engine
 
       /// Random number generator (must be initialized in init. list of c'tor!)
-      boost::variate_generator<boost::mt19937&, boost::uniform_int<> > rand_gen_;
+      std::mt19937 rand_gen_;
```

Comment:
I think we need to be very careful about the std::random stuff at is unfortunately not deterministically portable.
We had this problem before and the only solution was to not use std generators
https://www.reddit.com/r/cpp/comments/7i21sn/til_uniform_int_distribution_is_not_portable/

Reply:
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/CHEMISTRY/DecoyGenerator.h#L89